README.md

```diff
@@ -4,12 +4,20 @@ Helper library for packing ML apps to be published on Qrithm
 
 ## Installation
 
-Add `git+https://github.com/qrithm/paqr@master` to the requirements.txt file of your paq.
-
-In the future we may cut releases, but for now this is based off master.
+Run `pip install git+https://github.com/qrithm/paqr.git`
```

Comment:
This is presumably being done in the same virtualenv as eventually contains all the other deps to run the predictor and whose requirements get dumped into Docker?

Reply:
This may not be necessary. You can just install paqr in your base environment. If you use commands like paqr.run or paqr.publish, these will use the Dockerfile of the paq